lib/dry/system/component.rb

```diff
@@ -1,49 +1,95 @@
-require 'inflecto'
+require 'concurrent/map'
+
+require 'dry-equalizer'
+require 'dry/system/loader'
```

Comment:
Do we need a require for "dry/system/errors" here? When I required this file individually, I was setting errors loading our errors 😆 e.g. `NameError: uninitialized constant Dry::System::Component::InvalidComponentError`